http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -70,6 +62,19 @@ class GceInterpreter[F[_]: Parallel: ContextShift: Logger](
                                                     config.gceConfig.zoneName,
                                                     params.runtimeConfig.machineType)
 
+      // Determine is user persisent disk has been formatted before
+      isGceFormatted <- params.runtimeConfig match {
+        case x: RuntimeConfig.GceWithPdConfig =>
+          for {
+            diskId <- F.fromEither(
+              x.persistentDiskId.toRight(new RuntimeException("Missing diskId in the request. This should never happen"))
+            )
+          } yield {
+            persistentDiskQuery.getIsGceFormatted(diskId, ctx.now).transaction.map{ x => x.isDefined }
```

Comment:
`persistentDiskQuery.getIsGceFormatted(diskId, ctx.now).transaction` will give an `F[???]`, hence you need to put this in side `for`, yield can only have pure values

Reply:
yep, i just updated this and pushed